pytorch_lightning/callbacks/progress/base.py

```diff
@@ -213,6 +213,7 @@ def get_standard_metrics(trainer: "pl.Trainer", pl_module: "pl.LightningModule")
     if pl_module.truncated_bptt_steps > 0:
         items_dict["split_idx"] = trainer.fit_loop.split_idx
 
+    # TODO: Adapt for trainer.loggers
```

Comment:
`LoggerCollection` concatenates `logger.version` here, but does it make sense to do that here? I don't think it really makes sense to include `logger.version` at all in the progress bar.